redfish-core/lib/log_services.hpp

```diff
@@ -575,7 +573,11 @@ inline void
                 thisEntry["@odata.id"] = dumpPath + entryID;
                 thisEntry["Id"] = entryID;
                 thisEntry["EntryType"] = "Event";
-                thisEntry["Created"] = crow::utility::getDateTime(timestamp);
+                thisEntry["Created"] =
+                    crow::utility::getDateTimeUint(timestamp);
+                thisEntry["Created"] =
+                    crow::utility::getDateTimeUint(timestamp);
+                thisEntry["Name"] = dumpType + " Dump Entry";
```

Comment:
Populating "Name" is not in this PR scope. Is it coming from the upstream commit ?

Reply:
yes , this is coming from upstream commit .